public/index.html

```diff
@@ -93,31 +93,31 @@ <h2>Hovedstyret</h2>
             </div>
             <div class="board-name-and-mobile">
               <div>
-                <p><a href="mailto:leder@cyb.no">Thor M. K. Høgås</a></p>
+                <p><a href="mailto:leder@cyb.no">Fredrik Magnussen</a></p>
                 <div>Leder</div>
               </div>
               <div>
-                <p><a href="mailto:leder@cyb.no">Nicolas Harlem Eide</a></p>
+                <p><a href="mailto:leder@cyb.no">Martin Heggem</a></p>
                 <div>Nestleder</div>
               </div>
               <div>
-                <p><a href="mailto:kasserer@cyb.no">Tor-Aksel Solberg</a></p>
+                <p><a href="mailto:kasserer@cyb.no">N/A</a></p>
```

Comment:
Maziar Kosarifar ble etterfylt så og si med en gang, var det ikke slik?

Reply:
Har HS hatt et møte enda? Kan ikke se referatet noe sted.

Reply:
Fra #generelt i slack:
```
Lee Kvåle [8:28 PM]
 Forøvrig!! @maziark er ny kasserer!
```

Reply:
Må jo uansett fattes et vedtak som burde referatføres.

Reply:
Nå har ikke jeg sett protokollen, men så vidt jeg skjønte var det en runde til med de vervene som ikke ble fylt rett etter første valgrunde. Kan være jeg misforsto, men med tanke på hvor raskt etter det ble sagt trur jeg det er riktig?

Men er vel bedre å få høre det fra noen som var på generalforsamlingen/har sett protokollen.

Reply:
Det var ikke under generalforsamlingen at dette ble bestemt. Var rett etter under mat og øl. Siden HS da ikke har hatt et møte burde det nok ventes til de har et møte for å fatte vedtaket før man legger han til på nettsiden. Selv om de per lovene kanskje var vedtaksdyktige på generalforsamling så er det vel dårlig stil og vedta slikt uten noe referat.

Reply:
Hehe, ja det er jo noe annet.

Reply:
Maziar er ikke kasserer før nye HS er konstituert, og det *nye* styret har fattet et vedtak på det (etter at de selvsagt er konsitutert). Derfor er han heller ikke skrevet opp på denne til det er gjort :)

Reply:
Nåværende HS kan vel fatte vedtaket, og det ligger på dagsorden til neste møte. Burde kanskje lage en egen PR med Maziar til det vedtaket er fattet, da det virker som dette skjer før resten av påtroppende skal på nettsiden (møtet er imorgen)